.github/workflows/test_linux_exports.yml

```diff
@@ -0,0 +1,101 @@
+name: 🐧 Linux Exports Tests
+on:
+  workflow_call:
+    inputs:
+      godot-version:
+        type: string
+      jvm-version:
+        type: string
+
+concurrency:
+  group: ci-${{github.actor}}-${{github.head_ref || github.run_number}}-${{github.ref}}-linux_exports_tests
+  cancel-in-progress: true
+
+jobs:
+  test-linux-exports:
+    runs-on: ubuntu-20.04
+    name: ${{ matrix.name }}
+    strategy:
+      matrix:
+        include:
+          - name: dev tests
+            target: dev
+            bootstrap-target: debug
+          - name: release tests
+            target: release
+            bootstrap-target: release
+
+    steps:
```

Comment:
Do we even need to clone Godot here ? Everything is already built after all.

Reply:
The sources for the api gen ? Previously we needed Godot to get the xml documentation, but now even that is contained inside the api.json, a file that we directly commit to this repo. Piertho made the change, we no longer use the xml at all, it's all purely from the json.

Reply:
done